models/containerized_model/template.py

```diff
@@ -178,6 +178,10 @@ def generate_python(file_metadata):
             'prediction_service.py': {
                 'exec_permission': False,
                 'kwargs': {}
+            },
+            'requirements.txt': {
```

Comment:
this expects `requirements.txt` file to be present in the templates directory and without which will throw an error. I'll update this pr to add the file

Reply:
Thanks! I must have missed that in the commit